package.json

```diff
@@ -24,22 +24,22 @@
     "package": "yarn clean && yarn build"
   },
   "dependencies": {
-    "@aws-lambda-powertools/commons": "^1.18.0",
-    "@aws-lambda-powertools/logger": "^1.18.0",
-    "@aws-lambda-powertools/metrics": "^1.18.0",
-    "@aws-lambda-powertools/tracer": "^1.18.0",
-    "@aws-sdk/client-dynamodb": "^3.509.0",
-    "@aws-sdk/client-sqs": "^3.509.0",
-    "@aws-sdk/util-dynamodb": "^3.509.0",
-    "@smithy/node-http-handler": "^2.3.1"
+    "@aws-lambda-powertools/commons": "^1.18.1",
+    "@aws-lambda-powertools/logger": "^1.18.1",
+    "@aws-lambda-powertools/metrics": "^1.18.1",
+    "@aws-lambda-powertools/tracer": "^1.18.1",
+    "@aws-sdk/client-dynamodb": "^3.523.0",
+    "@aws-sdk/client-sqs": "^3.523.0",
```

Comment:
hey Liam - could you please build and deploy the application with the new dependencies and run the feature tests from you local machine to make sure that the uptick hasn't broken anything?